test/test_methods_test.rb

```diff
@@ -11,6 +11,18 @@ def test_asserts_controller_submits_correct_form
     end
   end
 
+  def test_asserts_controller_submits_at_all
+    runner = FooControllerTest.new :foo
+
+    error = assert_raises MiniTest::Assertion do
+      runner.assert_submits FooForm do
+      end
+    end
+
+    assert_equal "Expected TestMethodsTest::FooController to submit a TestMethodsTest::FooForm",
+                 error.message
```

Comment:
"Expected TestMethodsTest::FooController to submit a TestMethodsTest::FooForm" is 76 characters with spaces included, so maybe bump that down to 75? 

Yeah, I think that makes sense to me...I'm guessing there will be exceptions that will pop up later, but that is fine with me